dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequest.java

```diff
@@ -326,6 +326,11 @@ public void initForNextRequest(IRequest req) {
 
             this._request = req;
             _srtRequestHelper = getRequestHelper();
+            
```

Comment:
I assume these were not meant to be left in.

Reply:
Not sure what this comment refers to ... lines 323-336 look unchanged from the prior version of this class, and I do not recall any reason I would have for changing them.